### DISCUSSION TITLE
AI Codebase Chat + Documentation for Deno

### DISCUSSION THREAD

**Discussion by maxsieg** [denoland/deno#25105] [Show and tell]
Hi all,

I wanted to see if the Deno community finds our AI codebase chat and documentation tool useful.
You can try it here for free: https://wiki.mutable.ai/denoland/deno

Give the chat a try and let me know what you think! You can ask complex questions like:

- "Where is {something you are looking for} implemented?"
- "How would I add {new feature}?"
- "How is {some technology} used in this repo?"

The documentation comprehensively explains all of your code, and links directly to the source code it is explaining.

I would love to hear your feedback on whether this is a valuable resource for the Deno community! We are happy to keep the wiki up to date for free, and provide all users with 10 free chats per month - in exchange, we ask that you put our badge in your README (it looks like this:
[![Mutable.ai Auto Wiki]([https://img.shields.io/badge/Auto_Wiki-Mutable.ai-blue)]](https://img.shields.io/badge/Auto_Wiki-Mutable.ai-blue)](https://wiki.mutable.ai/denoland/deno)

Let me know what you all think.

**Comment by rahulmr** [comment]
Looks good, I was thinking in similar way but more of the kind bolt.new , where it will help create apps made in deno and use webcontainers. LLMs will support Deno 2.0

**Comment by iugo** [comment]
Thanks for sharing! I really like the idea of an AI tool that links documentation directly to source code — that can be very helpful for new contributors and maintainers. However, I tried the link you provided and mutable.ai is currently unavailable, so I couldn’t try or verify the tool.
